src/Microsoft.Framework.Logging/LoggerFactory.cs

```diff
@@ -10,11 +10,12 @@ namespace Microsoft.Framework.Logging
     /// <summary>
     /// Summary description for LoggerFactory
     /// </summary>
-    public class LoggerFactory : ILoggerFactory
+    public class LoggerFactory : ILoggerFactory, IDisposable
```

Comment:
ILoggerFactory here should inherit from IDisposable too. (FYI: I already have a PR out for disposing this factory in hosting https://github.com/aspnet/Hosting/pull/316)